### DISCUSSION TITLE
Future-ready talent self-learning modules problem

### DISCUSSION THREAD

**Discussion by kirangalande137** [github/india#849] [Students]
Future-ready talent self-learning modules do not get updated only show in progress but I completed them already.
and i got mail from future ready talent to you have not compeleted any azure course

**Comment by vssai** [comment]
i am facing the same problem as that its showing in progress even though i completed the modules is there any solution

**Reply by ayeshaumraoshaik** [reply]
Facing same problem...what to do now

**Comment by ayeshaumraoshaik** [comment]
I am also facing same problem
